### PR TITLE
Add cxs example.

### LINKED ISSUE #579: Add CXS example
https://github.com/jxnblk/cxs#server-side-rendering

### PR DESCRIPTION
**Pull request by aranajhonny**
fixes #579 

Using cxs for styling. (supports SSR as well)
Live demo --> https://with-cxs-uxfjhopktn.now.sh/

### CODE REVIEW

**Comment by rauchg** [comment]
With JavaScript disabled:

image

Amazing job @aranajhonny. Thanks for the perfect work and the demo.

**Comment by aranajhonny** [comment]
@rauchg no lo habia probado con js desactivado 👍

**Comment by rauchg** [comment]
Es una gran técnica para asegurarse 100% que todo esté perfecto 🙌

File: examples/with-cxs/pages/_document.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import Document, { Head, Main, NextScript } from 'next/document'
+import cxs from 'cxs'
+
+export default class MyDocument extends Document {
+  static async getInitialProps ({ renderPage }) {
+    const page = renderPage()
+    let style = cxs.getCss()
```
Comment (rauchg) [inline examples/with-cxs/pages/_document.js]: @jxnblk is there a `cxs.reset` command or similar, that would ensure to clear _dirty_, _unclaimed_ state (for example if `renderPage` throws here?

**Pull request merged**